tests/python_build/test_glsl_builder.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from glsl_builders import build_rd_header, build_raw_header
+
+
+@pytest.mark.parametrize(
+    ["shader_files", "builder"],
+    [
+        ("glsl/vertex_fragment", build_raw_header),
```

Comment:
Just the rd_glsl folder is needed here. The non-GLES3 GLSL builder won't be used for the GLES3-style shader files. So I would just remove the glsl/ folder and keep the gles3/ and rd_glsl/ folders as they are right now.

Reply:
According to build logs (example below), the function `build_raw_header` is used for shaders in the directory `modules/lightmapper_rd`. The dummy test shaders I put into `fixtures/glsl` are based on shader syntax from aforementioned directory, so from my point of view it all matches together. Am I missing some piece of information here?

example from log:
```
[Initial build] Building GLSL header: "modules/lightmapper_rd/lm_blendseams.glsl.gen.h"
```

link between the message `Building GLSL header` and `build_raw_header`:
https://github.com/godotengine/godot/blob/898e09e2e63c221b861cc64b12adc80dc48ce5eb/SConstruct#L764